spec/services/admin/collections_report_query_spec.rb

```diff
@@ -18,12 +18,15 @@
     collection.save!
     collection
   end
+  # rubocop:disable Metrics/LineLength
+  let!(:collection4) { create(:collection_version_with_collection, state: 'decommissioned', name: 'dCollection').collection }
+  # rubocop:enable Metrics/LineLength
```

Comment:
Could another option here be to use `let(...) do ... end`, which we use for collections 2 and 3 above (to reduce the rubocop noise here)?

Reply:
good idea;  will do.